features/git-sync/all_branches/non_feature_branch_rebase_tracking_branch_conflict/last_branch/without_open_changes.feature

```diff
@@ -1,27 +1,23 @@
 Feature: git sync --all: handling rebase conflicts between non-feature branch and its tracking branch without open changes
 
   Background:
-    Given I have branches named "production" and "qa"
```

Comment:
production is necessary to me here. This is showing proper skip behavior as its the last branch in a git sync all that hits a conflict